Spanner/src/Connection/Grpc.php

```diff
@@ -275,6 +294,74 @@ public function getInstanceConfig(array $args)
         ]);
     }
 
+    /**
+     * @param array $args
+     */
+    public function createInstanceConfig(array $args)
+    {
+        $instanceConfigName = $args['name'];
+
+        $instanceConfig = $this->instanceConfigObject($args, true);
+        $res = $this->send([$this->getInstanceAdminClient(), 'createInstanceConfig'], [
+            $this->pluck('projectName', $args),
+            $this->pluck('instanceConfigId', $args),
+            $instanceConfig,
+            $this->addResourcePrefixHeader($args, $instanceConfigName)
+        ]);
+
+        return $this->operationToArray($res, $this->serializer, $this->lroResponseMappers);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function updateInstanceConfig(array $args)
+    {
+        $instanceConfigName = $args['name'];
+
+        $instanceConfigArray = $this->instanceConfigArray($args);
+
+        $fieldMask = $this->fieldMask($instanceConfigArray);
+
+        $instanceConfigObject = $this->serializer->decodeMessage(new InstanceConfig(), $instanceConfigArray);
```

Comment:
Because:
1. We need to call `$instanceConfigArray = $this->instanceConfigArray($args);` to get the instance config as an array, as that is needed as the input for getting the field mask.
2. Calling `instanceConfigArray($args)` changes the `$args` array, so it cannot be used again for a call to `$this->instanceConfigObject(..);`

This function is following the same structure as the existing method for `updateInstance` and reusing the existing functions from that as much as possible.